packages/tasks/src/local-apps.ts

```diff
@@ -95,7 +95,8 @@ export const LOCAL_APPS = {
 		docsUrl: "https://lmstudio.ai",
 		mainTask: "text-generation",
 		displayOnModelPage: isGgufModel,
-		deeplink: (model) => new URL(`lmstudio://open_from_hf?model=${model.id}`),
+		deeplink: (model, filepath) =>
+			new URL(`lmstudio://open_from_hf?model=${model.id}` + filepath ? `&file=${filepath}` : ""),
```

Comment:
Awesome addition.  What would be the format of `filepath`?  `[filename].[ext]`?

Reply:
Yep sir, the full filepath (including any potential directory) inside the model repo

e.g. inside https://huggingface.co/bartowski/Codestral-22B-v0.1-GGUF it could be `Codestral-22B-v0.1-IQ2_M.gguf` or `Codestral-22B-v0.1-f32.gguf/Codestral-22B-v0.1-f32-00001-of-00003.gguf` for instance

Reply:
got it, thanks 👍